test/screenshot/text-field/variants.tsx

```diff
@@ -5,5 +5,4 @@ export default [
   'textArea',
   'refTest',
   'autoFocus',
-  'characterCounter'
```

Comment:
I actually think we wanted this but let's just move this into another PR after we get this one in.

Reply:
Ah then we don't need to create another PR, right?

Reply:
Correct :)